### PR TITLE
solved  #24  popup upgrade pro dialog

### PR DESCRIPTION
**Pull request by adilkadivala**
When you click the upgrade to pro button, it should open the dialog.

### CODE REVIEW

File: src/components/layouts/Layout.tsx [hunk 1]
```diff
@@ -15,6 +19,13 @@ const Layout = ({ main }: { main: React.ReactNode }) => {
 
         <LayoutApplicationSection>{main}</LayoutApplicationSection>
 
+        {/* Dialog for Pricing */}
+        <Dialog>
+          <DialogContent className="h-[90vh] w-[90vw] max-w-[600px]">
+            <Pricing />
+          </DialogContent>
+        </Dialog>
+
```
Comment (bryanlundberg) [inline src/components/layouts/Layout.tsx:28]: MMM, this code is not doing anything, or I'm wrong?

**Review by bryanlundberg — COMMENTED** [review]
It's almost ready, just verify the changes done on Layout.tsx file. 😅

**Comment by adilkadivala** [comment]
let me check,

**Comment by adilkadivala** [comment]
Oh !! yes , You are right 


<Dialog>
          <DialogContent className="h-[90vh] w-[90vw] max-w-[600px]">
            <Pricing />
          </DialogContent>
        </Dialog>
        
        this code is not doing anything, 
        
        
        
        sorry, I have removed it, and raised new branch
        
        Thanks for guiding me,  and teaching me contributing in open-source

**Comment by adilkadivala** [comment]
hey will you not merge my PR ?

**Review by bryanlundberg — APPROVED** [review]
@adilkadivala thanks!

**Pull request merged**